Formula/cromshell-alpha.rb

```diff
@@ -0,0 +1,55 @@
+class CromshellAlpha < Formula
+  include Language::Python::Virtualenv
+
+  desc "Python tool to interact with a Cromwell server"
+  homepage "https://github.com/broadinstitute/cromshell"
+  url "https://storage.googleapis.com/bshifaw-bucket/cromshell-alpha.2.0.0.tar.gz"
+  sha256 "bfbb00bdea6cfefb3061860d6c7563e609a56175d221f9876a62bafe4be67eb5"
+  license "BSD-3-Clause"
+  depends_on "python" => "3.7"
+
+  resource "termcolor" do
+      url "https://files.pythonhosted.org/packages/8a/48/a76be51647d0eb9f10e2a4511bf3ffb8cc1e6b14e9e4fab46173aa79f981/termcolor-1.1.0.tar.gz"
```

Comment:
Please tell me these are generated automatically somehow?  Can you add a comment with how so future maintainers (i.e. me) can know?

Reply:
Unfortunately, this was done manually and there doesn't seem to be any built-in brew options to create one for you. There is however this python package called [python-brewer](https://pypi.org/project/python-brewer/) that hasn't been updated since 2018 and doesn't run anything higher than py3.7 that creates formulae. 
which requires env to be downgraded to pipdeptree==1.0.0 and pip==18.0.0. I'll try to test it out later and add instructions here

Reply:
Have you tried ` brew update-python-resources` or homebrew-pypi-poet?  There's an [article here](https://docs.brew.sh/Python-for-Formula-Authors) that makes it sound like one of those should work.

Reply:
I'm having trouble running any of the above tools to get the resource stanza for cromshell all at once. They may need the tool to be sourced from pypi, for example this is the results from `brew update-python-resources`
```
(venv) ➜  brew-formula brew update-python-resources cromshell-alpha
Error: Could not infer PyPI package name from URL:
  https://storage.googleapis.com/bshifaw-bucket/cromshell-alpha.2.0.0.tar.gz
```
`homebrew-pypi-poet` doesn't work on cromshell either since its not on pypi, but its possible to run it on packages individually from pypi. So we're not manually going to the site and looking up the URL and sha256.
```
(venv) ➜  brew-formula poet click
  resource "click" do
    url "https://files.pythonhosted.org/packages/f4/09/ad003f1e3428017d1c3da4ccc9547591703ffea548626f47ec74509c5824/click-8.0.3.tar.gz"
    sha256 "410e932b050f5eed773c4cda94de75971c89cdb3155a72a0831139a79e5ecb5b"
  end
```